common/protos-go/common/access_control.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.0
```

Comment:
Avoid checking in `protos-go` unless you have made any changes to `common/protos` data structures. You can remove these from your commit